src/plugins/section/section.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * Sections Plugin
+ *
+ * Copyright 2019 Marc Schreiber (@schrieveslaach)
+ * Released under the MIT license.
+ */
+/* global document */
+
+( function( document ) {
+    "use strict";
+    let root, api, gc;
+
+    let indexedSteps = null;
+
+    const agenda = document.querySelector( "#impress-section-agenda" );
+    const currentSection = document.querySelector(
+        "#impress-section-overview .impress-current-section" );
+    const sectionNumbers = document.querySelector(
+        "#impress-section-overview .impress-section-numbers" );
```

Comment:
You should set these in the `impress:init` event. Some other plugin/js might generate html dynamically before this plugin.